### DISCUSSION TITLE
Change p7zip's upstream to maintained fork

### DISCUSSION THREAD

**Discussion by kidonng** [Homebrew/discussions#454] [Everyday usage]
https://github.com/jinfeihan57/p7zip is an active fork of the origianl p7zip. It incorporates many fixes and new features, notably zstd support.
Does it qualifies to become the new upstream of Homebrew's p7zip formula? As far as I know [Termux](https://github.com/termux/termux-packages/blob/master/packages/p7zip/build.sh) has been using the fork for some time.

**Comment by kidonng** [comment]
Tagging some maintainers here (sorry if it disturbs!) @carlocab @SMillerDev.
Or should I create an issue in Homebrew/homebrew-core instead?

**Reply by SMillerDev** [reply]
You should create a pull request. We don't really accept requests for changes as issues or discussions.

**Comment by kidonng** [comment]
@SMillerDev Thanks for explaining!
